test/extensions/filters/http/jwt_authn/jwks_async_fetcher_test.cc

```diff
@@ -193,7 +191,9 @@ TEST_P(JwksAsyncFetcherTest, TestGoodFetchAndRefresh) {
   EXPECT_EQ(out_jwks_array_.size(), 1);
 
   // Expect refresh timer is enabled.
-  EXPECT_CALL(*timer_, enableTimer(expected_duration_, nullptr));
+  const std::chrono::milliseconds expected_refetch_time =
+      JwksAsyncFetcher::getCacheDuration(config_) - std::chrono::seconds(5);
```

Comment:
should we create a local variable RefetchBeforeExpiredSec for `std::chrono::seconds(5)` ?
So that we know how that value of 5 is set.

Reply:
Done